prometheus/src/main/java/org/opensearch/sql/prometheus/functions/resolver/QueryExemplarsTableFunctionResolver.java

```diff
@@ -0,0 +1,48 @@
+package org.opensearch.sql.prometheus.functions.resolver;
+
+import static org.opensearch.sql.data.type.ExprCoreType.LONG;
+import static org.opensearch.sql.data.type.ExprCoreType.STRING;
+import static org.opensearch.sql.prometheus.utils.TableFunctionUtils.getNamedArgumentsOfTableFunction;
+import static org.opensearch.sql.prometheus.utils.TableFunctionUtils.validatePrometheusTableFunctionArguments;
+
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import org.apache.commons.lang3.tuple.Pair;
+import org.opensearch.sql.expression.Expression;
+import org.opensearch.sql.expression.function.FunctionBuilder;
+import org.opensearch.sql.expression.function.FunctionName;
+import org.opensearch.sql.expression.function.FunctionResolver;
+import org.opensearch.sql.expression.function.FunctionSignature;
+import org.opensearch.sql.prometheus.client.PrometheusClient;
+import org.opensearch.sql.prometheus.functions.implementation.QueryExemplarFunctionImplementation;
+
+@RequiredArgsConstructor
+public class QueryExemplarsTableFunctionResolver implements FunctionResolver {
```

Comment:
It is possible to use merge this resolver with `QueryRangeTableFunctionResolver`? The only difference is argument count and function name. Have a look to `RelevanceFunctionResolver` as a reference.

Reply:
I prefer not to merge them and keep it a class per functionality use case

Reply:
maybe extract common base resolver ?

Reply:
I have already extracted common code to Utils.  Merging into single class can have following problems in future. 
* All the relevance functions belong to same class and have similar properties. we are not sure of this in case of prometheus functions.
* argument validations of all functions will be in a single file.
* Same class will have details of all functions.[We need to maintain map of functionName with function details.]

Reply:
Can you add javadoc to this class please?